src/Idea.hs

```diff
@@ -90,16 +90,30 @@ idea severity hint from to = rawIdea severity hint (srcInfoSpan $ ann from) (f f
 idea' :: (GHC.HasSrcSpan a, Outputable.Outputable a) =>
          Severity -> String -> a -> a -> [Refactoring R.SrcSpan] -> Idea
 idea' severity hint from to =
-  rawIdea severity hint (ghcSpanToHSE (GHC.getLoc from)) (f from) (Just $ f to) []
-  where
-    f = Outputable.showSDocUnsafe . Outputable.ppr
```

Comment:
I also needed this in `ignoreNoSuggestion'`, so I moved it out. Is this fine? (+ is it fine to always rely on `showSDocUnsafe`?)